platform/android/export/export.h

```diff
@@ -31,6 +31,82 @@
 #ifndef ANDROID_EXPORT_H
 #define ANDROID_EXPORT_H
 
+#include <core/ustring.h>
+
 void register_android_exporter();
 
+String ANDROID_MANIFEST_TEXT = "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
```

Comment:
Add a TODO saying we want to find a better place for this, e.g. a templates directory